frontend/src/pages/MainPage.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
```

Comment:
In new versions of React (17+) you don't need to import React if you are not using it. It's more like suggestion than thing that needs to be changed, though webstorm doesn't like this.